hasura/graphql-engine
# server: fix #5542 - check for empty string in insert_event_log

I've just added an explicit check for the empty string. Not the most elegant solution, but I know exception handling would add a runtime cost, which is why we have two branches in the first place.

Note: I didn't fix this in the RQL code because we need to also be able to handle the case where a user performs an insert directly, without going via Hasura at all.

### Changelog


### Affected components



### Related Issues

Fixes #5542

### Steps to test and verify
I tested this in `psql` as follows:

```
begin;
set local hasura.tracecontext = '';
select hdb_catalog.insert_event_log('public', 'test', '', '', '{}');
...
rollback;
```

Before this change, you will see the reported exception. After this change, the trigger data will just be set to `NULL`.

### Server checklist


#### Catalog upgrade

#### Metadata
Does this PR add a new Metadata feature?
#### GraphQL
#### Breaking changes
